### ISSUE TITLE
Customer Service to support archival to DataONE through February 28, 2025

### ISSUE THREAD

**Issue by MathewBiddle** [ioos/marine_life_data_network#124]
### Who is requesting this?

Marine Life

### What is being requested?

- Finalize last step in ATN DAC [workflow](https://cdn.ioos.noaa.gov/media/2022/03/ATN-DAC-Data-Management-Policy-Guidance_v.202201.pdf) - DataONE - Continue pushing data packages to DataONE as they are finalized and ready for release
- Prioritized list from @mmckinzie 

### What is the requested deadline and why?

2025-02-28

### What is the current status quo (i.e., what happens if this does not get done)?

- Data not published publicly

### What indicates this is done (i.e., how do we know this is complete)?

- [ ] 4 projects finalized and released to DataONE

### Provide a description or any other important information.

It would be beneficial to understand who "owns" the different parts of this process. I think Meg does some of it, Axiom does other parts. Just something to keep in mind for when we start diving into the documentation efforts.

**Comment by MathewBiddle** [comment]
Moved to `In progress` as we should be working on this now.

**Comment by iwensu0313** [comment]
Update: no new datasets to archive to DataONE yet since Dec 1, 2024.

**Comment by mmckinzie** [comment]
@mmckinzie Will prioritize DataONE datasets in February. Two previously released datasets, whose dataset-level ISO 19115 metadata records will need to be updated to add relevant NCEI identifiers and rereleased at DataONE, also need to be updated at DataONE as capture in issue #125 .

**Comment by MathewBiddle** [comment]
any updates?

**Comment by mmckinzie** [comment]
still working on this

**Comment by MathewBiddle** [comment]
Where did this land?

**Comment by mmckinzie** [comment]
One project (294 deployments) was submitted to DataONE ((https://doi.org/10.24431/rw1k8er).

Two other projects are in the works.

**Comment by MathewBiddle** [comment]
Thanks! 

We will track the other two in another issue. 

Closing.

**Comment by MathewBiddle** [comment]
BTW, we were hoping for 3 projects. Only 1 was published. It would be good to have a TLDR; of why only one was published.

**Comment by mmckinzie** [comment]
Because I am still working with PIs on finalizing the other two projects. 

And I was also waiting on issues with the NCEI Archiver tool to be resolved and the pathway automated so I could resume flagging deployments for release to NCEI. It didn't make sense to amend DataONE projects metadata to include the ATN collection DOI and corresponding list of accession numbers until all deployments under a particular project were submitted to NCEI. This applies to 5 projects that have already been released to DataONE and one of the projects that is in progress. 

Pushing datasets/updates to DataONE metadata is 'low' effort by Axiom and can likely resume under March1+ activities.

**Comment by mmckinzie** [comment]
It's generally a bit of both. Every project is unique. 

In this particular case, one of the projects (NAVFAC harbor seals) is more or less ready, I just need to find time to do the final review of the DataONE dataset/metadata and flag for release. But that project also has data files that will go to NCEI (processed/filtered data to DataONE, raw data to NCEI), so I was waiting until the issues with the NCEI Archiver we're resolved so I could release those files first and then add corresponding identifiers to the dataset-level record for DataONE. I also still need to make newly required modifications to this project's deployment metadata in the ADR to ensure NCEI files are generated correctly (part of the solution to handle re-used ptt ids) and implement the work-around to the project-level metadata to ensure the creator/contributor fields in NCEI files are properly populated which has to be reverted after NCEI files are generated so I don't like to do that step until I'm ready to flag files for release.  

For the other project (MBA dolphinfish), I am waiting for PIs to finalize their dataset level metadata record within the RW, this process generally takes a fair amount of back and forth to ensure records are complete and well described.  I also just received missing deployment-level metadata from PIs late last week which I haven't had a chance to review yet and upload to the RW. But based on email communications I also know these deployment-level metadata are not complete and that I'll need to work with the manufacturers on the PIs behalf to try and recover missing metadata for select deployments. Once the deployment and dataset level metadata are finalized this project should also be ready for release. 

It typically takes Axiom a low amount of effort to release DataONE datasets once everything is staged and ready to go, but it generally requires med/high level of effort for me and PIs to prep datasets for release.
